organizing/README.md

```diff
@@ -2,9 +2,57 @@
 
 
 ## Event organizing
-TODO add context inline and revise of https://docs.google.com/document/d/195b7COKkuYSjd6QQ58ji6KhzoXpTZA58XFR5Wg2oStk/edit
-- feedback forms - https://docs.google.com/document/d/1mYdS9KvE75-OFBZ3dcgpQtw6L6qbUNkTJBX40ei-PL0/edit
-- introduction slides example - https://docs.google.com/presentation/d/15ou4tBs845m10InGNDslNn5cYWlH-CHxDrlFF4kesNE/edit?usp=sharing
+
+### Finding host
+We have many hosts reaching out to us in the email. We are contacting them 2-3 months before to book the date. We also have a list of past hosts in Trello when we need to ask explicitely. Reach out to them with the [Host initial email](host.md).
```

Comment:
We have many hosts reaching out to us by email (I think). Also explicitly without the e